demo

```diff
@@ -105,6 +105,15 @@ demo.test() {
     local clustertasks=$@
   fi
 
+  OSArch=$(uname -m)
+  if [ "$OSArch" == "ppc64le" ]; then
+          sed -i 's/gcr.io\/cloud-builders\/mvn@sha256:57523fc43394d6d9d2414ee8d1c85ed7a13460cbb268c3cd16d28cfb3859e641/maven:3.6.3-adoptopenjdk-8/' ./maven/pipeline.yaml
+          sed -i 's/gcr.io\/cloud-builders\/mvn@sha256:57523fc43394d6d9d2414ee8d1c85ed7a13460cbb268c3cd16d28cfb3859e641/maven:3.6.3-adoptopenjdk-8/' ./jib-maven/pipeline.yaml
+  elif [ "$OSArch" == "s390x" ]; then
+          sed -i 's/gcr.io\/cloud-builders\/mvn@sha256:57523fc43394d6d9d2414ee8d1c85ed7a13460cbb268c3cd16d28cfb3859e641/maven:3.6.3-adoptopenjdk-11/' ./maven/pipeline.yaml
+          sed -i 's/gcr.io\/cloud-builders\/mvn@sha256:57523fc43394d6d9d2414ee8d1c85ed7a13460cbb268c3cd16d28cfb3859e641/maven:3.6.3-adoptopenjdk-11/' ./jib-maven/pipeline.yaml
```

Comment:
Where did you get `maven:3.6.3-adoptopenjdk-11` from? Are we going to recommend this image to our customers?